spec/support/helpers/authentication.rb

```diff
@@ -0,0 +1,17 @@
+module Helpers
+  module Authentication
+    def log_in_as(user)
+      visit new_user_session_path
+
+      within "#new_user" do
+        fill_in 'user[email]', with: user.email
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.